client/packages/programs/src/RnRForms/DetailView/AppBarButtons.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+  AppBarButtonsPortal,
+  DownloadIcon,
+  EnvUtils,
+  Grid,
+  LoadingButton,
+  Platform,
+  PrinterIcon,
+  ReportContext,
+  useParams,
+  useTranslation,
+} from '@openmsupply-client/common';
+import {
+  ReportRowFragment,
+  ReportSelector,
+  usePrintReport,
+} from '@openmsupply-client/system';
+import { useRnRForm } from '../../api';
+import { JsonData } from '@openmsupply-client/programs';
+
+export const AppBarButtonsComponent = () => {
+  const { id = '' } = useParams();
+  const { data /* isLoading */ } = useRnRForm({ rnrFormId: id });
+  const { print, isPrinting } = usePrintReport();
+  const t = useTranslation();
+
+  const printReport = (
+    report: ReportRowFragment,
+    args: JsonData | undefined
+  ) => {
+    if (!data) return;
+    print({
+      reportId: report.id,
+      dataId: data?.id,
+      args,
+    });
+  };
+
+  return (
+    <AppBarButtonsPortal>
+      <Grid container gap={1}>
+        <ReportSelector
+          context={ReportContext.Requisition}
```

Comment:
Is it a Requisition or will this need its own report context?

Reply:
Would be better to have its own context - otherwise the report would be available when you print a requisition / internal order